pom.xml

```diff
@@ -465,6 +466,13 @@
                 <classifier>all</classifier>
             </dependency>
 
+            <dependency>
+                <groupId>com.databend</groupId>
+                <artifactId>databend-jdbc</artifactId>
+                <version>0.3.2</version>
```

Comment:
The latest version 0.3.4.

Reply:
Done.

Reply:
The v0.3.4 has some bugs and wait it to be fixed then upgrade.

Reply:
Please elaborate on "some bugs".